test/integration/scheduler_perf/config/performance-config.yaml

```diff
@@ -1,264 +1,377 @@
-      podTemplatePath: config/pod-with-node-affinity.yaml
-  params:
-  - numNodes: 500
-    numInitPods: [500]
-    numPodsToSchedule: 1000
-  - numNodes: 5000
-    numInitPods: [5000]
-    numPodsToSchedule: 1000
-- template:
-    desc: TopologySpreading
-    nodes:
-      nodeTemplatePath: config/node-default.yaml
-      labelNodePrepareStrategy:
-        labelKey: "topology.kubernetes.io/zone"
-        labelValues: ["moon-1", "moon-2", "moon-3"]
-    initPods:
-    - podTemplatePath: config/pod-default.yaml
-    podsToSchedule:
-      podTemplatePath: config/pod-with-topology-spreading.yaml
-  params:
-  - numNodes: 500
-    numInitPods: [1000]
-    numPodsToSchedule: 1000
-  - numNodes: 5000
-    numInitPods: [5000]
-    numPodsToSchedule: 2000
-- template:
-    desc: PreferredTopologySpreading
-    nodes:
-      nodeTemplatePath: config/node-default.yaml
-      labelNodePrepareStrategy:
-        labelKey: "topology.kubernetes.io/zone"
-        labelValues: ["moon-1", "moon-2", "moon-3"]
-    initPods:
-    - podTemplatePath: config/pod-default.yaml
-    podsToSchedule:
-      podTemplatePath: config/pod-with-preferred-topology-spreading.yaml
-  params:
-  - numNodes: 500
-    numInitPods: [1000]
-    numPodsToSchedule: 1000
-  - numNodes: 5000
-    numInitPods: [5000]
-    numPodsToSchedule: 2000
-- template:
-    desc: MixedSchedulingBasePod
-    nodes:
-      nodeTemplatePath: config/node-default.yaml
-      labelNodePrepareStrategy:
-        labelKey: "topology.kubernetes.io/zone"
-        labelValues: ["zone1"]
-    initPods:
-    - podTemplatePath: config/pod-default.yaml
-    - podTemplatePath: config/pod-with-pod-affinity.yaml
-    - podTemplatePath: config/pod-with-pod-anti-affinity.yaml
-    - podTemplatePath: config/pod-with-preferred-pod-affinity.yaml
-    - podTemplatePath: config/pod-with-preferred-pod-anti-affinity.yaml
-    podsToSchedule:
-      podTemplatePath: config/pod-default.yaml
-  params:
-  - numNodes: 500
-    numInitPods: [200, 200, 200, 200, 200]
-    numPodsToSchedule: 1000
-  - numNodes: 5000
-    numInitPods: [2000, 2000, 2000, 2000, 2000]
-    numPodsToSchedule: 1000
-- template:
-    desc: Preemption
-    initPods:
-    - podTemplatePath: config/pod-low-priority.yaml
-    podsToSchedule:
-      podTemplatePath: config/pod-high-priority.yaml
-  params:
-  - numNodes: 500
-    numInitPods: [2000]
-    numPodsToSchedule: 500
-  - numNodes: 5000
-    numInitPods: [20000]
-    numPodsToSchedule: 5000
-- template:
-    desc: Unschedulable
-    skipWaitUntilInitPodsScheduled: true
-    initPods:
-    - podTemplatePath: config/pod-large-cpu.yaml
-    podsToSchedule:
-      podTemplatePath: config/pod-default.yaml
-  params:
-  - numNodes: 500
-    numInitPods: [200]
-    numPodsToSchedule: 1000
-  - numNodes: 5000
-    numInitPods: [200]
-    numPodsToSchedule: 5000
-  - numNodes: 5000
-    numInitPods: [2000]
-    numPodsToSchedule: 5000
+- name: SchedulingBasic
+  workloadTemplate:
+  - createNodes: $initNodes
```

Comment:
I would prefer something like:

```
- op: createNodes
  number: $initNodes
```

Then you can use the string to disambiguate when unmarshaling

Reply:
Done.